ground/src/test/java/com/google/android/ground/ui/map/gms/mog/MogSourceTest.kt

```diff
@@ -24,24 +24,24 @@ import org.mockito.junit.MockitoJUnitRunner
 
 @RunWith(MockitoJUnitRunner::class)
 class MogSourceTest {
-  private val mogSource = MogSource("url/{x}/{y}.tif", 5..7)
+  private val mogSource = MogSource( 5..7, "url/{x}/{y}.tif")
```

Comment:
Nit: Extra space before "5"

Reply:
Thank you! I need to figure out how to automatically run the linter after every code change. Done.

Reply:
I think you can do something like this in AS. YMMV:

<img width="1094" alt="image" src="https://github.com/google/ground-android/assets/228050/dad7821a-b2b9-402e-a7e0-fc85cf2de12d">